schema/index.js

```diff
@@ -11,7 +14,26 @@ const resolvers = {};
   });
 });
 
-let rootSchema = '';
+resolvers.ObjID = new GraphQLScalarType({
+  name: 'ObjID',
+  description: 'Id representation, based on Mongo Object Ids',
+  parseValue(value) {
+    return ObjectId(value); // eslint-disable-line new-cap
```

Comment:
Are we planning on including a lint file in the output? I'd say just disable the rule. CRA has the opinion of only including rules that check for actual errors by default, which is a reasonable approach IMO.